packages/hdwallet-portis/src/portis.ts

```diff
@@ -251,8 +190,10 @@ export class PortisHDWallet implements HDWallet, ETHWallet {
           console.error(e)
       }
       for (let i = 0; i < msg.length; i++) {
-        const { addressNList } = msg[i];
-        const portisResult = await this.portis.getExtendedPublicKey(addressNListToBIP32(addressNList))
+        const { addressNList, coin } = msg[i];
+        const slip44 = 0x80000000 + slip44ByCoin(coin)
+        // TODO we really shouldnt be every using the "bitcoin" string parameter but is here for now to make it work with their btc address on their portis wallet.
+        const portisResult = await this.portis.getExtendedPublicKey(addressNListToBIP32(addressNList), addressNList[1] === slip44 ? 'Bitcoin' : '')
```

Comment:
What I suggested here may have broken it for ETH, since `slip44ByCoin('Ethereum')` will give you `60'`, and that will match when given a standard ETH path, causing the ternay to say `Bitcoin`.... which IIUC, isn't what you want.

Reply:
I think what you want instead is `const slip44 = slip44ByCoin('Bitcoin')`

Reply:
how about this:

```
        const bitcoinSlip44 = 0x80000000 + slip44ByCoin('Bitcoin')
        const portisResult = await this.portis.getExtendedPublicKey(addressNListToBIP32(addressNList), addressNList[1] === bitcoinSlip44 ? 'Bitcoin' : '')
```

Reply:
ya.  that

Reply:
perfect